src/main/protobuf/BUILD

```diff
@@ -15,6 +15,19 @@ load("@rules_java//java:defs.bzl", "java_import")
 
 package(default_visibility = ["//visibility:public"])
 
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("@rules_java//java:defs.bzl", "java_proto_library")
+
+alias(
+    name = "deps_java_proto",
+    actual = "@deps//:java_proto",
```

Comment:
What is @deps, and should it have a less-likely-to-collide-with-users'-naming-choices name?

Reply:
It's the http_java_proto repo for the deps proto. Only available with the development downloads, so I would expect that it would only collide after voiding the warranty.

Reply:
Ahhh.  gotcha.